pyproject.toml

```diff
@@ -9,6 +9,7 @@ dependencies = [
     "opentelemetry-api>=1.20.0",
     "opentelemetry-sdk>=1.20.0",
     "opentelemetry-exporter-otlp-proto-http>=1.20.0",
+    "tiktoken>=0.5.1"
```

Comment:
Optional dependency may be unfriendly to  development. 
Optional dependency will be carefully designed in the near future.